packages/code-studio/src/settings/FormattingSectionContent.jsx

```diff
@@ -820,12 +820,18 @@ export class FormattingSectionContent extends PureComponent {
             Applies a formatting rule to all columns of a set type.
           </div>
           <div className="form-row mb-2">
-            <label className="col-form-label col-3">Time zone</label>
+            <label
+              className="col-form-label col-3"
+              htmlFor="select-reset-timezone"
+            >
+              Time zone
+            </label>
             <div className="col pr-0">
               <select
                 className="custom-select"
                 value={timeZone}
                 onChange={this.handleTimeZoneChange}
+                id="select-reset-timezone"
```

Comment:
Fine here since there's only ever one `FormattingSectionContent`, but as a side note if you need to add the `id` anywhere else - if it's in any component that could be re-used, you should give it a unique ID (generate a shortid on component creation and use it in the element ID)